bottlecap/src/config/mod.rs

```diff
@@ -33,6 +34,13 @@ pub struct Config {
     pub apm_enabled: bool,
     pub lambda_handler: String,
     pub serverless_flush_strategy: FlushStrategy,
+    pub trace_enabled: bool,
+    pub serverless_trace_enabled: bool,
+    pub capture_lambda_payload: bool,
```

Comment:
I was gonna say should we remove this? Then I remembered that we are currently targeting Node + Python